### ISSUE TITLE
Set language of result download

### ISSUE THREAD

**Issue by lauraykerr** [CDCgov/prime-simplereport#4007]
## Area or Component
Result download 

## Quick Summary
The result download PDF doesn't have a set language (metadata). 

## Accessibility Violation
PDFs should have a primary language set so screen readers can read them properly. 

## Action Requested
If possible, English result downloads should have primary language set to English. Spanish results should have language set to Spanish.

**Comment by emyl3** [comment]
@lauraykerr what is the best way to check this? Is it by going to Document Properties > Advanced > under "Reading Options" > "Language" ?

**Comment by lauraykerr** [comment]
@emyl3 Yes! The screenshot is the way I've seen to do it.

**Comment by emyl3** [comment]
Awesome -- thank you for confirming with me. I'll need to do some research if that is something we can pass from our end when we generate the pdf! :)

**Comment by emyl3** [comment]
Used this library https://github.com/diegomura/react-pdf as it had compatibility with react and  lots of activity while also appearing as the go-to library for this kind of effort. It was also not the easiest to style and also threw a bunch of warnings in IntelliJ.

This was able to add the title and language to the pdf. HOWEVER!! It resolved these two issues but introduced like 10 more that were more serious in severity. The effort to use this and the drawbacks of this library was not worth adding this fix. A search of their issues shows their  current accessibility issues: https://github.com/diegomura/react-pdf/search?q=accessibility&type=issues

Closing this ticket.
